storage/gc_queue.go

```diff
@@ -219,12 +219,9 @@ func (gcq *gcQueue) process(now proto.Timestamp, rng *Replica) error {
 	switch len(gcArgs.Keys) {
```

Comment:
Remove the switch statement in favor of:

    if len(gcArgs.Keys) == 0 {
      return nil
    }
    .... // all the stuff under "default"